### PR TITLE
updating caption on my slide

### LINKED ISSUE #1: Add collaborators to this repo
## Comment below to be added as a collaborator

We will all be using this repository today. Even though it's public right now, you will not be able to make changes until you're given the correct permissions. We've automated this process via the GitHub API. Once you comment, we will add you as a collaborator.

You'll start to receive a lot of emails. (❗) You can slow these down immediately by clicking the **Unwatch** button at the top of the page and selecting either the **Participating and @mentions** or **Ignore** option.

### PR DESCRIPTION
**Pull request by cody-carlson-learning**
This PR updates the copy of my slide because it was out of date

This PR closes / fixes / resolves #1 

## Done
- made a commit
- created a branch

## To Do
- [ ] merge the branch
- [ ] delete the branch afterwards
- [ ] add 3rd task for demo purposes

### CODE REVIEW

**Review by Daduc38 — APPROVED** [review]
Code has been properly made. Good Work 👍

File: _slides/03-cody-carlson-learning.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hey what a great caption!
```
Comment (cody-carlson) [inline _slides/03-cody-carlson-learning.md]: Look great! 🏆

**Review by cody-carlson — COMMENTED** [review]
ok sounds good

File: _slides/03-cody-carlson-learning.md (hunk 1 above)
Comment (cody-carlson-learning) [reply _slides/03-cody-carlson-learning.md]: thanks for the feedback!

**Pull request merged**